src/App.vue

```diff
@@ -50,6 +42,7 @@ watch(modalOpen, val => {
 
 <template>
   <div
+    v-if="skinName"
```

Comment:
I think we shouldn't have this, this create situation where we have blank screen